cri/v1alpha2/cri_utils_test.go

```diff
@@ -841,7 +878,30 @@ func Test_getUserFromImageUser(t *testing.T) {
 		want  *int64
 		want1 string
 	}{
-	// TODO: Add test cases.
+		{
+			name: "Empty Test",
+			args: args{
+				imageUser: "",
+			},
+			want:  nil,
```

Comment:
We should check whether the value is equal other then the pointer.

Reply:
Thanks for your comment. All above mentioned has been done.